### DISCUSSION TITLE
Quick idea and announcement!

### DISCUSSION THREAD

**Discussion by Benji377** [Benji377/OpenMusic#103] [Ideas]
We are slowly but surely approaching version 1.2.0 and I think we should make it something special.
Here are the criteria to make the new 1.2.0 release:
- Design change is finished and approved
- Full app refactoring completed
- Playlist functionality added
- All known bugs till now fixed

The reason that this release is so special and requires special criteria is that I would like to publish the app to the Google Play store as soon as the new release gets out. This will get us new users and hopefully new contributors with new feedback and ideas.

**Comment by Benji377** [comment]
Now there are some things I would like to check with you @squivix first. As you may have noticed we are technically the only one working on this project right now and since there is no private messaging on Github I will write you here:
First of all, do you like the new player fragment? And are you ok with us bringing the app to the play store? Should we keep it free or charge something?

Now, here is what I think:
I like the new design. I especially like that the user can now pick two colors as background. I would just like to get rid of the 10 seconds forward or rewind buttons and add a function to the previous and next buttons that when you long press on them they slowly go 10 seconds forward or rewind in the song. I think that it might be a really cool and unique experience to publish an app. I have never done it before and I am super excited. In my opinion, we should also keep the app free, maybe add a Patreon link where users can sponsor us. It's an open-source app, after all, everyone can copy the code if they want.

**Reply by squivix** [reply]
>First of all, do you like the new player fragment?

I like the idea of letting the user pick the color too but I don't think the gradient looks as good as the background we had before. So I like everything new besides the background.

>And are you ok with us bringing the app to the play store?

Yes I'm fine with it. I think the app is still under-development but if you want users and user feedback that sounds good.

>Should we keep it free or charge something?

Up to you, but personally I don't think too many people would pay for a music player. I know I wouldn't so I vote to keep it free. I'm working on this to improve my android development skills and to have some experience with version control systems, so I'm already benefiting from contributing.

Good luck with the new version!

**Reply by Benji377** [reply]
I thought about it again and I think we should wait till the 1.3 release since the 1.2 release itself will be pretty huge
